CHANGELOG.md

```diff
@@ -152,10 +152,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     - Added `postgres-migration-async-operation` lambda to start an ECS task to run a the `data-migration2` lambda.
     - Updated `async_operations` table to include `Data Migration 2` as a new `operation_type`.
     - Updated `cumulus-tf/variables.tf` to include `optional_dynamo_tables` that will be merged with `dynamo_tables`.
+  - **CUMULUS-2453**
+    - Created `storeErrors` function which stores errors in system bucket.
+    - Updated `executions` and `granulesAndFiles` to call `storeErrors` to store migration errors.
+    - Added `system_bucket` variable to `data-migration2`.
   - **CUMULUS-2451**
     - Added summary type file `packages/db/src/types/summary.ts` with `MigrationSummary` and `DataMigration1` and `DataMigration2` types.
     - Updated `data-migration1` and `data-migration2` lambdas to return `MigrationSummary` objects.
     - Added logging for every batch of 100 records processed for executions, granules and files, and PDRs.
+    - Removed `RecordAlreadyMigrated` logs in `data-migration1` and `data-migration2`
```

Comment:
This line seems to duplicate the one below it?

Reply:
[`c6c1f32` (#2179)](https://github.com/nasa/cumulus/pull/2179/commits/c6c1f32eff6ceb5882a6dbb716014d89e99b8866#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL164)